curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md

```diff
@@ -7,14 +7,19 @@ dashedName: step-68
 
 # --description--
 
-One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` element as a child of the `head` element. Set its `charset` attribute to `UTF-8`.
+You can set the browser behavior by using `meta` element. A famous use case is telling the browser to parse the markdown into multiple languages by setting `charset` attribute to `UTF-8`.
```

Comment:
````suggestion
You can set browser behavior by adding self-closing `meta` elements in the `head`. Here's an example:

```
<meta attribute="value" />
```
````

Reply:
> and if they still use the closing tag, then just let it pass.

I'm not saying this would be the end of the world, but I don't think the HTML specs allow for a closing `</meta>` tag and the W3 HTML validator throws an error if you have one. My understanding is that we want to allow valid HTML wherever possible. Conversely, I think we should fail invalid HTML as well.

Reply:
We've had a lot of back and forth on the `/>` for self-closing elements. Pretty sure we don't need the `/` and shouldn't include in the example?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
> Pretty sure we don't need the / and shouldn't include in the example?

I'm fine with either method here.